pkg/crc/preflight/preflight_common.go

```diff
@@ -37,17 +37,16 @@ func preflightCheckAndFix(configuredToSkip bool, check, fix PreflightCheckFixFun
 	if ok {
 		return
 	}
-
-	if configuredToWarn {
-		logging.Warn(err.Error())
-		return
-	}
 	logging.Debug(err.Error())
 
 	ok, err = fix()
 	if ok {
 		return
 	}
 
+	if configuredToWarn {
+		logging.Warn(err.Error())
+		return
+	}
```

Comment:
@anjannath are those changes are as expected in the other preflight side like mac, linux ?